app/src/main/java/org/simple/clinic/teleconsultlog/drugduration/DrugDurationSheet.kt

```diff
@@ -69,7 +69,7 @@ class DrugDurationSheet : BottomSheetActivity(), DrugDurationUi, DrugDurationUiA
 
     MobiusDelegate.forActivity(
         events = events.ofType(),
-        defaultModel = DrugDurationModel.create(drugDuration.duration),
+        defaultModel = DrugDurationModel.create(drugDuration.duration!!),
```

Comment:
Yes, it's possible because we pass the frequency that is present in `PrescribedDrug`. 

I have a question regarding this, we can make it as a required value, and pass a default value when constructing the `DrugDuration`. That way we these screens won't know about any default values and just display whatever we sent.  WDYT?

Reply:
Makes sense, seems like a good idea.